jt-threshold/src/main/java/it/geosolutions/jaiext/threshold/ThresholdCRIF.java

```diff
@@ -0,0 +1,66 @@
+/* JAI-Ext - OpenSource Java Advanced Image Extensions Library
+ *    http://www.geo-solutions.it/
+ *    Copyright 2016 GeoSolutions
+
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+
+ * http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package it.geosolutions.jaiext.threshold;
+
+import it.geosolutions.jaiext.range.Range;
+
+import java.awt.RenderingHints;
+import java.awt.geom.AffineTransform;
+import java.awt.image.RenderedImage;
+import java.awt.image.renderable.ParameterBlock;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.media.jai.CRIFImpl;
+import javax.media.jai.ImageLayout;
+import javax.media.jai.ROI;
+
+import com.sun.media.jai.opimage.RIFUtil;
+
+/**
+ * A <code>CRIF</code> supporting the "Clamp" operation on rendered and renderable images.
+ */
```

Comment:
Make sure to update the name of the operation in the comment. It still says "clamp"